src/main/java/org/bitbuckets/drive/controlsds/DriveControlSDSSetup.java

```diff
@@ -167,8 +173,9 @@ public DriveControlSDS build(ProcessPath path) {
         double OrientKP = Preferences.getDouble(DriveSDSConstants.kOrientPKey, DriveSDSConstants.kOrientkP);
         double OrientKI = Preferences.getDouble(DriveSDSConstants.kOrientIKey, DriveSDSConstants.kOrientkI);
         double OrientKD = Preferences.getDouble(DriveSDSConstants.kOrientDKey, DriveSDSConstants.kOrientkD);
+        double OrientKF = Preferences.getDouble(DriveSDSConstants.kOrientFKey, DriveSDSConstants.kOrientkF);
 
-        PIDController rotControllerRad = new PIDController(OrientKP, OrientKI, OrientKD);
+        ProfiledPIDFController rotControllerRad = new ProfiledPIDFController(OrientKP, OrientKI, OrientKD, OrientKF,new TrapezoidProfile.Constraints(DriveSDSConstants.MAX_ANG_VELOCITY,DriveSDSConstants.MAX_ANG_VELOCITY));
```

Comment:
Here, you're setting max acceleration as max velocity. Make new constants in preferences for max auto orient angular velocity and acceleration.